internal/local/tls.go

```diff
@@ -160,6 +157,7 @@ func (service *ProvisioningService) ConfigureTLSSettings(setting tls.SettingData
 	log.Trace(putResponse)
```

Comment:
lets remove the trace message here as well

Reply:
done